rabbitmq-quickstart/rabbitmq-quickstart-processor/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+# Configure the incoming RabbitMQ queue `quote-requests`
```

Comment:
As we don't have dev services for RabbitMQ I would add the broker URL just to make it explicit.

Reply:
The referenced pull request pull request in Quarkus adds Dev Services support